lib/sequenceserver/blast/formatter.rb

```diff
@@ -39,10 +43,12 @@ def run
         command =
           "blast_formatter -archive '#{archive_file}'" \
           " -outfmt '#{format} #{specifiers}'" \
-          " -out '#{file}' 2> /dev/null"
+          " -out '#{file}' 2> #{tmp.path}"
         logger.debug("Executing: #{command}")
         Dir.chdir(File.exist?(DOTDIR) && DOTDIR || Dir.pwd) do
           system(command)
+          status = $CHILD_STATUS.exitstatus
+          fail ArgumentError, "Error in report generation" if status != 0
```

Comment:
Need a more informative error message. I would show the stderr to user. And since this is for #194, I would additionally add "Maybe it's because sequence ids are not unique across database. See - https://groups.google.com/d/msg/sequenceserver/QbE11EMQoEo/nAcCOmlsSjwJ"

Also, did you verify if checking for exit status is enough to catch the issue? Going through mailing list archives indicates blast_formatter returns 0 in this case. And thus we must check if there's something on stderr or what.